### ISSUE TITLE
can't import typing.OrderedDict with an alias

### ISSUE THREAD

**Issue by brahn** [microsoft/pyright#864]
**Describe the bug**
Pyright objects to aliasing of `typing.OrderedDict`.  Doesn't seem to occur with (for example) `typing.Tuple`

**To Reproduce**

```python
from collections import OrderedDict
from typing import Tuple as TupleAlias, OrderedDict as OrderedDictAlias

# no warning
foo: TupleAlias[int, str] = (1, "two")

# warning: Expected class type but received "_Alias"
bar: OrderedDictAlias[int, str] = OrderedDict() 
```

<img width="652" alt="scratch py — data-infra 2020-07-20 13-58-05" src="https://user-images.githubusercontent.com/235302/87985801-10413900-ca91-11ea-8f38-592d87c093a4.png">

**Expected behavior**
No warning

**VS Code extension or command-line**
VSCode extension v1.1.54

**Comment by erictraut** [comment]
Thanks for reporting. I hadn't noticed that `typing.OrderedDict` was introduced in Python 3.7 as a generic version of `collections.OrderedDict`. Should be easy to fix.

**Comment by brahn** [comment]
Thank you for the quick response!

**Comment by erictraut** [comment]
This is now implemented in Pyright 1.1.55, which I just published. It will also be included in the next release of Pylance.

**Comment by brahn** [comment]
> This is now implemented in Pyright 1.1.55, which I just published. It will also be included in the next release of Pylance.

Amazing! Thanks so much

**Comment by brahn** [comment]
(just verified, works as expected! 🎉 )
